data/json/items/book/traps.json

```diff
@@ -74,7 +74,7 @@
     "symbol": "?",
     "color": "green",
     "skill": "traps",
-    "max_level": 3,
```

Comment:
I would suggest that if you are making the max 5 you need a minimum level of around 3.  Which might mean there needs to be a book that handles the lower level of traps.

Reply:
Makes sense, done!

"The Modern Trapper" book might already fill that niche? It sort of makes sense to start with making bear traps and then getting into explosive traps.

Reply:
Although I'll need to tweak the skill values in the practice recipes to account for this... And do some more testing :)

Reply:
I've added a basic military trap book!